app-emulation/winetricks/winetricks-20170614.ebuild

```diff
@@ -21,7 +21,7 @@ SRC_URI="${SRC_URI}
 	kde? ( https://dev.gentoo.org/~tetromino/distfiles/wine/${wtg}.tar.bz2 )"
 
 DESCRIPTION="Easy way to install DLLs needed to work around problems in Wine"
-HOMEPAGE="http://winetricks.org http://wiki.winehq.org/winetricks"
```

Comment:
I'll add the Wiki page back in to HOMEPAGE just to err on the side of having more information available.